apps/ts-minbar-test-react-components/project.json

```diff
@@ -2,8 +2,5 @@
   "name": "ts-minbar-test-react-components",
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32277&commitId=c17a68895aaa1fcc317a0346f5f0624883af934b&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|26| Changed|
|Avatar Converged.badgeMask.chromium.png|69| Changed|

</details>


  <details><summary><b>Drawer</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|1136| Changed|
|Drawer.Full Overlay Dark Mode.chromium.png|975| Changed|
|Drawer.Full Overlay RTL.chromium.png|1153| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>